tests/test_web_start_bot.py

```
@@ -0,0 +1,70 @@
+    assert updated_stages["message"] == expected_message
+
+def test_translations_task_stops_when_no_new_translations(monkeypatch, tmp_path):
+    stages = {"status": None, "message": None, "sub_name": None}
+
+    dummy_main_path = tmp_path / "downloads"
+    dummy_main_path.mkdir()
+
+    fake_svg_path = dummy_main_path / "Example.svg"
+    fake_svg_path.write_text("<svg></svg>")
+
+    def fake_download_one_file(title, out_dir, index):
```

Ruff (0.14.2)

Unused function argument: `title`

(ARG001)